optional_plugins/varianter_yaml_to_mux/tests/test_mux.py

```diff
@@ -228,9 +228,11 @@ def test_fingerprint_order(self):
         variant1 = next(iter(mux1))
         variant2 = next(iter(mux2))
         self.assertNotEqual(variant1, variant2)
-        self.assertEqual(str(variant1), "{'paths': '', 'variant': "
-                         "[TreeNode(name='child1'), TreeNode(name="
-                         "'child2')], 'variant_id': 'child1-child2-9154'}")
+        str_variant = str(variant1)
+        self.assertIn("'paths': ''",  str_variant)
+        self.assertIn(("'variant': [TreeNode(name='child1'), "
+                       "TreeNode(name='child2')]"), str_variant)
+        self.assertIn("'variant_id': 'child1-child2-9154'", str_variant)
```

Comment:
The new test decreases the coverage. Missing part is to make sure there are no other items than the expected ones. My suggestion is to use something like this:
```diff
--- a/optional_plugins/varianter_yaml_to_mux/tests/test_mux.py
+++ b/optional_plugins/varianter_yaml_to_mux/tests/test_mux.py
@@ -228,9 +228,17 @@ class TestMuxTree(unittest.TestCase):
         variant1 = next(iter(mux1))
         variant2 = next(iter(mux2))
         self.assertNotEqual(variant1, variant2)
-        self.assertEqual(str(variant1), "{'paths': '', 'variant': "
-                         "[TreeNode(name='child1'), TreeNode(name="
-                         "'child2')], 'variant_id': 'child1-child2-9154'}")
+        variant_list = []
+        for item in variant1:
+            variant_list.append("'%s': '%s'" % (item, variant1[item]))
+        expected_items = ["'paths': ''",
+                          "'variant': '[TreeNode(name='child1'), "
+                          "TreeNode(name='child2')]'",
+                          "'variant_id': 'child1-child2-9154'"]
+        for item in expected_items:
+            self.assertIn(item, variant_list)
+            variant_list.remove(item)
+        self.assertFalse(variant_list)
```

Reply:
Yes, makes sense.